OneTrustConsentFilter/.github/workflows/snapshot_release.yml

```diff
@@ -0,0 +1,40 @@
+name: Snapshot
+
+on:
+  pull_request:
+    branches: [ 'main' ]
+    types: [ 'opened', 'reopened', 'edited', 'synchronize' ]
+
+jobs:
+  snapshot:
+    runs-on: ubuntu-latest
+    environment: deployment
+    steps:
+      - name: Checkout source branch
+        uses: actions/checkout@v3
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x OneTrustConsentFilter/gradlew
+
+      - name: cache gradle dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-core-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-core-
+
+      - name: Publish Onetrust snapshot to sonatype
+        run: ./OneTrustConsentFilter/gradlew publishToSonatype
+        env:
+          SIGNING_KEY_ID: ${{ secrets.SIGNING_KEY_ID }}
+          SIGNING_KEY_PASSWORD: ${{ secrets.SIGNING_KEY_PASSWORD }}
+
+          ORG_GRADLE_PROJECT_sonatypeUsername: ${{ secrets.NEXUS_USERNAME }}
```

Comment:
have we added al these secrets in the repo settings?

Reply:
No, these are  organization wide secrets.